### DISCUSSION TITLE
request from android client is blocked

### DISCUSSION THREAD

**Discussion by 0xori** [invertase/react-native-firebase#4431] [Q&A]
I'm developing a new RN app (not in the stores yet), trying to implement login with phone number.
I followed the instructions from https://rnfirebase.io/#installation and https://rnfirebase.io/auth/phone-auth .
also added the sha1 to firebase console.
and I'm getting this error:
[error: [auth/unknown] an internal error has occurred. [ requests from this android client application fun.together.solo are blocked. ]]

tried to reinstall, clean android build, fresh new react-native-init app, nothing helps. any idea?

**Comment by mikehardy** [comment]
Hi there! if it is an emulator, you have to use testing numbers, they don't work on the emulator. For phone auth in general you need to test on a real device. I'm guessing that's it?

**Comment by 0xori** [comment]
No, I'm testing on a real device
